### ISSUE TITLE
pgo show backups failing

### ISSUE THREAD

**Issue by alrooney** [CrunchyData/postgres-operator#2266]
pgo show backups is failing with following error.  Do not know why.

```
$ pgoc pgo show backup retroelk-prod1-azure -n pgo
Error: primary pod is not in Ready state
command terminated with exit code 2
alec@Alecs-MacBook-Pro ~/Developer/violetatrium/nginx/nginx_frontend (MIN-8209) $ pgoc pgo show cluster retroelk-prod1-azure -n pgo

cluster : retroelk-prod1-azure (crunchy-postgres-gis-ha:centos7-11.9-2.5-4.5.0)
	pod : retroelk-prod1-azure-cmyc-55fd469bdc-p6z7g (Running) on prod1-db-0 (2/2) (replica)
		pvc: retroelk-prod1-azure-cmyc (100G)
	pod : retroelk-prod1-azure-siaf-7f96d774b6-kp9t4 (Running) on prod1-db-3 (2/2) (primary)
		pvc: retroelk-prod1-azure-siaf (100G)
	pod : retroelk-prod1-azure-tapu-67bc49c74b-v4n28 (Running) on prod1-db-2 (2/2) (replica)
		pvc: retroelk-prod1-azure-tapu (100G)
	resources : CPU: 8 Memory: 16Gi
	limits : CPU: 15 Memory: 48Gi
	deployment : retroelk-prod1-azure-backrest-shared-repo
	deployment : retroelk-prod1-azure-cmyc
	deployment : retroelk-prod1-azure-pgbouncer
	deployment : retroelk-prod1-azure-siaf
	deployment : retroelk-prod1-azure-tapu
	service : retroelk-prod1-azure - ClusterIP (10.105.129.54)
	service : retroelk-prod1-azure-pgbouncer - ClusterIP (10.101.227.65)
	service : retroelk-prod1-azure-replica - ClusterIP (10.98.138.154)
	pgreplica : retroelk-prod1-azure-cmyc
	pgreplica : retroelk-prod1-azure-siaf
	pgreplica : retroelk-prod1-azure-tapu
	labels : pgouser=admin service-type=NodePort pgo-version=4.5.0 autofail=true crunchy-pgbadger=false crunchy-postgres-exporter=true pg-pod-anti-affinity= NodeLabelValue=postgres deployment-name=retroelk-prod1-azure-siaf pgo-backrest=true workflowid=4098ec34-1913-48e8-86a6-b46690c41030 crunchy-pgha-scope=retroelk-prod1-azure custom-config=retroelk-custom-config name=retroelk-prod1-azure pg-cluster=retroelk-prod1-azure 
```

**Comment by jkatz** [comment]
Reported in #1927 and resolved in 4.5.1.

**Comment by alrooney** [comment]
Thanks!
